common/notification/actions.ts

```diff
@@ -503,6 +503,14 @@ const _notificationActions = {
             redirectTo: '/start',
         },
     },
+    student_login: {
+        description: 'Student logged in',
+        sampleContext: {},
+    },
+    pupil_login: {
+        description: 'Pupil logged in',
+        sampleContext: {},
+    },
```

Comment:
I'm not sure if we should create dedicated events for login with PW and login with Token

Reply:
Don't think we need it